nilearn/glm/first_level/first_level.py

```diff
@@ -220,6 +220,38 @@ def run_glm(Y, X, noise_model='ar1', bins=100,
         results = {0.0: ols_result}
 
     return labels, results
+  
+  
+def _check_trial_type(events):
```

Comment:
Not sure what was decided for type hints. `List[PathLike]` would be better in my opinion, but we can also do:

```suggestion
def _check_trial_type(events: list) -> None:
```

Reply:
OK, then I suppose having the simple list annotation as in the suggestion would be better for now.